week2/nikolay-r/tapac12.py

```diff
@@ -0,0 +1,11 @@
+import re
+
+
+def text_stats(text):
+    '''Receives random text and returns statistics about it.'''
+    capital_letters = len(re.findall('([A-Z][a-z]+)', text))
+    words_count = len(re.findall('(\w+)', text))
+    sentences_count = len(re.findall('([A-Z][^\.!?]*[\.!?])', text))
+    occurences_of_and = len(re.findall(r'\band\b', text))
+
+    return capital_letters, words_count, sentences_count, occurences_of_and
```

Comment:
why not return a dict with these values?

Reply:
```
In [9]: result = text_stats('Mishkata si ima pishka, a pishkata and and  Mishka.')
In [10]: result
Out[10]: (2, 9, 1, 2)
```

No test.
What if I use any other argument different than str. How you would handle that?
No explanation what is what in the result. As @EmiliyanPanagyurski suggested, please use dictionary to explain.